src/Visualizer/public/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
@llFreetimell do we need license comment for html ?

Reply:
I don't think it's necessary.
The basic structure of all HTML files looks like above.

Reply:
Only saying conclusion, everything is okay :)

For convention with other files, adding the license comments seems better.
But it is not mandatory because we already have `LICENSE` and `COPYRIGHT` files at the root directory of the repository.